### DISCUSSION TITLE
The icon in the title can't show

### DISCUSSION THREAD

**Discussion by MoLiShang** [nuxt/nuxt#8874] [Questions (Nuxt 2)]
When i set the meta in nux.config.js  with local link,the icon can't show 
      { name: 'viewport', content: 'width=device-width, initial-scale=1', rel: 'icon', href: '/favicon.ico' },
If i set the remote link,the icon can show.
  { name: 'viewport', content: 'width=device-width, initial-scale=1', rel: 'icon', href: 'http://xxx.com/favicon.ico' },
But icon link awlays in local,what could i do to show it?

**Comment by danielroe** [comment]
It looks like you're combining a link and a meta tag. Can you confirm what your actual config is, and whether you have a favicon in your static directory?

**Reply by MoLiShang** [reply]
This is my directory structure.
![image](https://user-images.githubusercontent.com/37530126/108787122-098f1d00-75b0-11eb-96e0-6106adecb0c4.png)
As you see,i have two favicon.ico in two direction,and i just want to ensure the config can read the link.
And i really test lot of  times in my browser,i can find the meta tag in the dom,And when i copy the icon link address,it also can show in the browser.
![image](https://user-images.githubusercontent.com/37530126/108787473-d39e6880-75b0-11eb-8d68-0665872a8216.png)
![image](https://user-images.githubusercontent.com/37530126/108787905-b918bf00-75b1-11eb-98af-7e4d0211183b.png)

**Reply by MoLiShang** [reply]
Then i want to see how native nuxt deal with it,but when i create a new nuxt app,it sounds like nuxt also have this problem。
![image](https://user-images.githubusercontent.com/37530126/108788491-119c8c00-75b3-11eb-8de8-765ca8bdb27a.png)
![image](https://user-images.githubusercontent.com/37530126/108788476-06e1f700-75b3-11eb-9973-ed5c91689516.png)
It sounds like native nuxt also can't show the title icon

**Reply by danielroe** [reply]
@MoLiShang As I said previously, you have an invalid entry for your `favicon` which is why it is not showing. It should not be:
```js
meta: [
  { name: 'viewport', content: 'width=device-width, initial-scale=1', rel: 'icon', href: 'http://xxx.com/favicon.ico' }
]
```
You have combined two different tags. Instead it should be as in the new Nuxt project you created.
```js
link: [
  { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
]
```

You can find out more [in the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link).

**Reply by MoLiShang** [reply]
I've done as your said ，but it not useful.Actually,my new test nuxt project also can't show.(just created,not change any code)And the pictures my last reply is belong my test project.

![image](https://user-images.githubusercontent.com/37530126/108841005-12183f80-7612-11eb-94b0-35616cbcc6e7.png)

I think the entry is not invalid because I can see the icon from the browser,the path i copied in the DOM is useful.
